### PR TITLE
major update of alta website

### PR DESCRIPTION
**Pull request by dainlp**
Change the UI and delete some out-of-date contents.

### CODE REVIEW

**Comment by dainlp** [comment]
major update of alta website
update brochure

File: index.html [hunk 1]
```diff
@@ -6,6 +6,40 @@
 <h1>Latest ALTA News</h1>
 
 <dl>
+    <dt> <strong>ALTA 2019 Workshop</strong> </dt>
+
+    <dd>
+        <p> The <a href="https://alta2019.alta.asn.au/">
+	          2019 Australasian Language Technology Association Workshop</a>
+            (ALTA 2019) was held from the 4th-6th of December at the
```
Comment (admackin) [inline index.html]:
```suggestion
            (ALTA 2019) will be held from the 4th-6th of December at the
```

File: _layouts/home.html [hunk 2]
```diff
@@ -114,36 +114,31 @@
...
    <body>
- <!-- This table contains the main structure of the page --> 
+ <!-- This table contains the main structure of the page -->
+<div class="pagehead">
+     <div>Australasian Language Technology Association</div>
+</div>
+
+ <div class="navigation">
+     <div>
+         <a style="float: left; height: 50px; padding: 15px 25px; font-size: 18px; line-height: 20px; text-transform: uppercase; color: #eee" href="index.html">Home</a>
```
Comment (admackin) [inline _layouts/home.html]: as above

**Review by admackin — CHANGES_REQUESTED** [review]
Looks great! Just a couple of minor things

**Comment by daixiangau** [comment]
updated

**Comment by dainlp** [comment]
I fix the typo of home page; put these styles in the css file.

**Pull request merged**